src/EnergyPlus/WindowManager.cc

```diff
@@ -928,7 +924,6 @@ namespace WindowManager {
                     AngleNum(IGlass) = LayerNum(IGlass);
                 }
             }
-            SetCommonIncidentAngles(ConstrNum, NGlass, TotalIPhi, AngleNum);
```

Comment:
If this function is removed, my understanding is that you force users to input the same incident angles in 3 tables. It reduces flexibility intended previously.

Reply:
No. The user doesn't have to use the same incident angles for any table. The tables are used to evaluate optical performance at any angle regardless of what the user inputs. There is no need to perform this intermediate interpolation...that's what `CurveValue(...)` does.